cartographer/mapping/map_builder.cc

```diff
@@ -227,12 +251,20 @@ void MapBuilder::SerializeState(io::ProtoStreamWriter* const writer) {
 }
 
 void MapBuilder::LoadMap(io::ProtoStreamReader* const reader) {
-  proto::PoseGraph pose_graph;
-  CHECK(reader->ReadProto(&pose_graph));
+  proto::PoseGraph pose_graph_proto;
+  CHECK(reader->ReadProto(&pose_graph_proto));
```

Comment:
I think it would make sense to introduce a pbstream serialization header proto, which would include the pose graph and trajectory builder options protos.